tests/001-test-use-devs-to-create-thin-pool.sh

```diff
@@ -5,9 +5,9 @@ test_devs() {
   local devs=$TEST_DEVS
   local test_status=1
   local testname=`basename "$0"`
-  local vg_name="dss-test-foo"
+  local vg_name="css-test-foo"
```

Comment:
I see that there are two kind of changes in this PR. One which are necessary and the one which are not. For example, this one is not necessary. Can we take all un-necessary changes and put in a separate PR and push that one first.